node/src/actors/chain_manager/actor.rs

```diff
@@ -90,6 +90,8 @@ impl ChainManager {
                 // Get consensus parameter from config
                 act.consensus_c = config.connections.consensus_c;
 
+                let _removed_transactions = act.transactions_pool.set_total_weight_limit(config.mining.transactions_pool_total_weight_limit);
```

Comment:
What about logging the evicted transactions at debug level?

Reply:
:+1:
Here `removed_transactions` should always be empty, but I added logs when inserting new transactions.